packages/frontend/src/redux/slices/nft/index.js

```diff
@@ -74,6 +78,17 @@ const fetchOwnedNFTsForContract = createAsyncThunk(
     }
 );
 
+const transferNFT = createAsyncThunk(
+    `${SLICE_NAME}/transferNFT`,
+    async ({ accountId, contractName, nft }, { dispatch, getState }) => {
+        debugLog('THUNK/transferNFT');
+
+        const { actions: { transferToken } } = nftSlice;
+        dispatch(transferToken({ contractName, nft }));
```

Comment:
We can refetch the token here rather than cache the transferred token object, but there isn't much interaction after the transfer so it may not be strictly necessary.